src/main/webapp/encounters/socialJson.jsp

```diff
@@ -99,9 +99,12 @@ String filter="SELECT FROM org.ecocean.MarkedIndividual where encounters.contain
 
 String individualIDFilter="";
 if(request.getParameter("individualID")!=null){
-	individualIDFilter = request.getParameter("individualID");
+	//individualIDFilter = request.getParameter("individualID");
+	individualIDFilter=" && enc.individual.individualID == '"+request.getParameter("individualID")+"'";
+	
 	//filter="SELECT FROM org.ecocean.MarkedIndividual where individualID == '"+request.getParameter("individualID")+"' || ((rel.individual1 == this || rel.individual2==this) && (rel.markedIndividualName1 =='"+request.getParameter("individualID")+"' || rel.markedIndividualName2 =='"+request.getParameter("individualID")+"')) VARIABLES org.ecocean.social.Relationship rel";
-	filter="SELECT FROM org.ecocean.MarkedIndividual where individualID == '"+request.getParameter("individualID")+"' || ((rel.individual1 == this && rel.markedIndividualName2 =='"+request.getParameter("individualID")+"') || (rel.individual2==this && rel.markedIndividualName1 =='"+request.getParameter("individualID")+"')) VARIABLES org.ecocean.social.Relationship rel";
+	//filter="SELECT FROM org.ecocean.MarkedIndividual where individualID == '"+request.getParameter("individualID")+"' || ((rel.individual1 == this && rel.markedIndividualName2 =='"+request.getParameter("individualID")+"') || (rel.individual2==this && rel.markedIndividualName1 =='"+request.getParameter("individualID")+"')) VARIABLES org.ecocean.social.Relationship rel";
```

Comment:
Might make sense to remove the commented-out code? IMO this is a good practice to get into the habit of. If we need it, it's always in older commits, right?